knowledge-base/customize-chart-tooltip-net-maui.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Customize ChartTooltipBehavior in Chart for .NET MAUI
```

Comment:
```suggestion
title: Customize the Behavior of the Chart Tooltip
```

Reply:
Rephrased the whole title as the article does not show how to customize the behavior of the tooltip, but rather the text and appearance.